spec/feature_tests/user_stories_spec.rb

```diff
@@ -0,0 +1,60 @@
+require 'takeaway'
+
+describe "User stories" do
+
+let(:takeaway) {Takeaway.new(double(:text, send: nil))}
+let(:menu) {"starter : 5\nmains : 10\ndesert : 3\n"}
+
+  # As a customer
+  # So that I can check if I want to order something
+  # I would like to see a list of dishes with prices
+  describe 'User Story 1' do
+
+    it 'allows a customer to see a list of dishes with prices' do
+      expect{takeaway.show_dishes}.to output(menu).to_stdout
+    end
+
+  end
+
+  # As a customer
+  # So that I can order the meal I want
+  # I would like to be able to select some number of several available dishes
+  describe 'User Story 2' do
+
+    it 'allows a customer to place an order' do
+      expect{takeaway.order("starter", 1, "mains", 2, "desert", 3, 34)}.not_to raise_error
+    end
+
+    it 'prevents a customer from ordering unavailable dishes' do
+      message = "Unable to place order: dish not on menu"
+      expect{takeaway.order("steak", 1, "mains", 2, "desert", 3, 34)}.to raise_error message
+    end
+  end
+
+  # As a customer
+  # So that I can verify that my order is correct
+  # I would like to check that the total I have been given matches the sum of the various dishes in my order
+  describe 'User Story 3' do
+
+    it 'raises error if customer total does not match the actual total' do
+      message = "Unable to place order: supplied total doesn't match actual total"
```

Comment:
Metrics/LineLength: Line is too long. [82/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)